Mathlib/Tactic/Linter/DeprecatedSyntaxLinter.lean

```diff
@@ -62,11 +62,40 @@ register_option linter.style.admit : Bool := {
   descr := "enable the admit linter"
 }
 
+/-- The option `linter.style.maxHeartbeats` of the deprecated syntax linter flags usages of
+`set_option maxHeartbeats n in cmd` that do not add a comment explaining the reason for the
```

Comment:
Can you update this comment to mention synthInstance.maxHeartbeats also?

Reply:
Done.